main.cpp

```diff
@@ -496,6 +501,8 @@ bool llama_model_load(const std::string & fname, llama_model & model, gpt_vocab
         fin.close();
     }
 
+    free(f_buf);
```

Comment:
`f_buf` will not be free if this function returns earlier, but I think it does not matter since it's a small amount of memory :)